.github/workflows/release_build.yaml

```diff
@@ -97,8 +97,10 @@ jobs:
           key: ${{ runner.os }}-tools-${{ github.sha }}
           restore-keys: |
             ${{ runner.os }}-tools-
+      # This is the same as the PR build workflow except that provide the
+      # version number via the TAG envvar (without the "v" prefix).
       - name: Build artifacts
-        run: ./.github/workflows/scripts/build_artifacts.sh
+        run: TAG="${GITHUB_REF#refs/tags/v}" ./.github/workflows/scripts/build_artifacts.sh
```

Comment:
Looking at `build_artifacts.sh`, one of the first things we do is `export TAG=` which I think will wipe this out? It then proceeds to use the `GITHUB_REF` env var to try an detect if the current build ref is a tag or not ... based on that, I don't think this is what we want? Perhaps something is wrong with `GITHUB_REF` of the tag matcher?

Reply:
ugg. i was looking at ./script/build_artifact.sh instead of the script in the workflows directory. Let me revisit.

Reply:
Yeah, the matcher presumes a leading / which doesn't exist. Will fix.